### ISSUE TITLE
Version 2.5.0 throws error in 'Configure System' section

### ISSUE THREAD

**Issue by RicardoArgg** [jenkinsci/xray-connector-plugin#50]
### Version report

Jenkins and plugins versions report:

```
Jenkins: 2.302
OS: Windows 10 - 10.0
---
ace-editor:1.1
ansicolor:1.0.0
ant:1.11
antisamy-markup-formatter:2.1
apache-httpcomponents-client-4-api:4.5.13-1.0
bootstrap4-api:4.6.0-3
bootstrap5-api:5.0.2-1
bouncycastle-api:2.20
branch-api:2.6.5
build-timeout:1.20
caffeine-api:2.9.1-23.v51c4e2c879c8
checks-api:1.7.2
cloudbees-folder:6.15
command-launcher:1.6
credentials-binding:1.27
credentials:2.5
display-url-api:2.3.5
durable-task:1.38
echarts-api:5.1.2-3
email-ext:2.83
font-awesome-api:5.15.3-4
gatling:1.3.0
git-client:3.8.0
git-server:1.10
git:4.7.2
github-api:1.123
github-branch-source:2.11.1
github:1.33.1
gradle:1.37.1
handlebars:3.0.8
jackson2-api:2.12.4
javadoc:1.6
jdk-tool:1.5
jira-steps:1.6.0
jjwt-api:0.11.2-9.c8b45b8bb173
jquery:1.12.4-1
jquery3-api:3.6.0-2
jsch:0.1.55.2
junit:1.51
ldap:2.7
lockable-resources:2.11
mailer:1.34
matrix-auth:2.6.7
matrix-project:1.19
maven-plugin:3.12
momentjs:1.1.1
okhttp-api:3.14.9
pam-auth:1.6
pipeline-build-step:2.14
pipeline-github-lib:1.0
pipeline-graph-analysis:1.11
pipeline-input-step:2.12
pipeline-milestone-step:1.3.2
pipeline-model-api:1.8.5
pipeline-model-definition:1.8.5
pipeline-model-extensions:1.8.5
pipeline-rest-api:2.19
pipeline-stage-step:2.5
pipeline-stage-tags-metadata:1.8.5
pipeline-stage-view:2.19
plain-credentials:1.7
plugin-util-api:2.4.0
popper-api:1.16.1-2
popper2-api:2.5.4-3
resource-disposer:0.16
role-strategy:3.1.1
scm-api:2.6.4
script-security:1.77
snakeyaml-api:1.29.1
ssh-credentials:1.19
ssh-slaves:1.32.0
sshd:3.1.0
structs:1.23
timestamper:1.13
token-macro:2.15
trilead-api:1.0.13
workflow-aggregator:2.6
workflow-api:2.46
workflow-basic-steps:2.23
workflow-cps-global-lib:2.21
workflow-cps:2.93
workflow-durable-task-step:2.39
workflow-job:2.41
workflow-multibranch:2.26
workflow-scm-step:2.13
workflow-step-api:2.24
workflow-support:3.8
ws-cleanup:0.39
xray-connector:2.5.0
```

- What Operating System are you using (both controller, and any agents involved in the problem)?

```
Windows 10
```

### Reproduction steps

- Step 1: Install the plugin  'Xray - Test Management for Jira' version 2.5.0 and restart
- Step 2: Go to 'Manage Jenkins' and click on 'Configure System'
- Step 3: Modify anything in the configurations and click on 'Save'

### Results

Expected result:

Configurations are saved correctly

Actual result:

I get an error 'Oops!, A problem occurred while processing the request'
Logging ID: 6f915532-09ee-4419-9b97-1f9d57cbec6b

from jenkins.err.log:
2021-07-20 19:32:59.718+0000 [id=46]	WARNING	o.e.j.s.h.ContextHandler$Context#log: Error while serving http://localhost:808/configSubmit
net.sf.json.JSONException: JSONObject["serverInstances"] is not a JSONArray.

[jenkins.err.log](https://github.com/jenkinsci/xray-connector-plugin/files/6851033/jenkins.err.log)

**Comment by Russell616** [comment]
Hi @RicardoArgg!

Is this a brand new instalation of the plugin? Or is it an upgrade?

**Comment by RicardoArgg** [comment]
it is a new installation

**Comment by jfernand21** [comment]
I hace the same issue. I was can

**Comment by Russell616** [comment]
@jfernand21 @RicardoArgg I just open a new PR that should fix this problem. It would be a great help if you installed the new plugin version (2.5.1 RC1) just to double check that the problem is indeed fixed

link to download RC1: https://ci.jenkins.io/job/Plugins/job/xray-connector-plugin/job/PR-51/1/artifact/target/xray-connector.hpi

**Comment by jfernand21** [comment]
The PR-51 it works and i could save the config as expected.

**Comment by Russell616** [comment]
Thank you for your feedback @jfernand21. I just released the plugin version 2.5.1 into the Jenkins plugin index, however, it may take a few hours to appear available in their website and to download into Jenkins.

Here it is the official published .hpi file if you want to download it now: https://repo.jenkins-ci.org/releases/org/jenkins-ci/plugins/xray-connector/2.5.1/xray-connector-2.5.1.hpi

I will now close this issue, thank you everyone for the help!
